test/Spec.hs

```diff
@@ -1,4 +1,10 @@
 module Main (main) where
 
+import Test.Hspec
+import Config
+import Crocodealer.Config
```

Comment:
According to our style guide, all imports should have explicit import lists. And also let's add empty line between dependencies from Hackage and dependencies from this package.